configuration/configuration_reference.md

```diff
@@ -313,13 +310,42 @@ The `<security>` element can be used to enable authentication. If the element is
 ```xml
 <server artifactsdir="/var/lib/go/big-artifacts-folder">
   <security allowOnlyKnownUsersToLogin="false">
-    <ldap uri="ldap://xxx.yourcompany.com"
-       managerDn="cn=Acitivity Directory LDap User,ou=InformationSystems,ou=SharedAccounts,ou=Principal,dc=xxxx,dc=yyyy,dc=com"
-       managerPassword="password"
-       searchFilter="(sAMAccountName={0})" />
-        <bases>
-          <base value="ou=Employees,ou=Enterprise,ou=Principal,dc=xxxx,dc=yyyy,dc=com"/>
-        </bases>
+    <authConfigs>
+     <authConfig id="profile-id" pluginId="cd.go.authentication.ldap">
+       <property>
+         <key>Url</key>
+         <value>ldap://xxx.yourcompany.com</value>
+       </property>
+       <property>
+         <key>ManagerDN</key>
+         <value>cn=Acitivity Directory LDap User,ou=InformationSystems,ou=SharedAccounts,ou=Principal,dc=xxxx,dc=yyyy,dc=com</value>
+       </property>
+       <property>
+         <key>Password</key>
+         <value>secret</value>
```

Comment:
Users are allowed to use both `value` and `encrypted_value`, infact if they were changing it, they would set `value` and not `encrypted_value`. So, I think its better to keep it to password. The plugin's docs should mention that `Password` is a secure property so server would encrypt it before saving.